build.gradle

```diff
@@ -675,11 +829,11 @@ task genFakeSource(dependsOn: ["buildYarnTiny", "mapNamedJar"]) {
 	}
 }
 
-task decompileCFR(type: JavaExec, dependsOn: "mapNamedJar") {
+task decompileCFR(type: JavaExec, dependsOn: [unpickNamedJar]) {
 	classpath = configurations.decompileClasspath
 	main = "org.benf.cfr.reader.Main"
 
-	args namedJar.getAbsolutePath(), "--outputdir", file("namedSrc").absolutePath
+	args unpickedJar.getAbsolutePath(), "--outputdir", file("namedSrc").absolutePath
```

Comment:
Won't this result in the JAR used in a development environment containing unpicked bytecode?
Early in Unpick's development it was pointed out to me that application of Unpick should only affect the generated sources, otherwise it breaks Mixin's `@ModifyConstant` in dev.

Reply:
No this is just yarn and the basic decompile task. The loom changes are separate and still to come.

Reply:
What's it decompiling & unpicking, if not the mapped MC JAR?

Reply:
It is, but this is in yarn and not loom/dev environments

Reply:
Oh right, for enigma